.github/workflows/release.yml

```diff
@@ -42,7 +42,10 @@ jobs:
           _links="${_links//$'\n'/'%0A'}"
           _links="${_links//$'\r'/'%0D'}"
           # Set output 'links' to $_links
-          echo "::set-output name=links::${_links}"
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
```

Comment:
are we confident this can't ever generate a string that would be invalid?

Reply:
This is how they recommend it's done, so I sure hope so. Prior to what's in the docs now they were showing it done like so:

```bash
DELIMITER=$(uuidgen)
echo "links<<$DELIMITER" >> "$GITHUB_OUTPUT"
echo "$_links" >> "$GITHUB_OUTPUT"
echo "$DELIMITER" >> "$GITHUB_OUTPUT"
```

I've been doing it both ways with changelog content & json blobs and haven't had any issues yet.

Here's the PR where this was added https://github.com/github/docs/pull/22463 and the original comment where `uuidgen` was suggested (shows up in another sample on the same page still) https://github.com/github/docs/pull/15354#issuecomment-1055867931.

Reply:
uuidgen seems simpler than this, but fair enough either way

Reply:
Yea, I'm not sure which way is "better" which is why I have both in my repos now 😆